src/analysis/normed_space/continuous_affine_map.lean

```diff
@@ -6,6 +6,7 @@ Authors: Oliver Nash
 import topology.algebra.continuous_affine_map
 import analysis.normed_space.add_torsor
 import analysis.normed_space.affine_isometry
+import analysis.normed_space.operator_norm
 
 /-!
 # Continuous affine maps between normed spaces.
```

Comment:
Can you mention your additions in the file-level docstring?

Reply:
Done.